lib/src/components/sidedrawer/SidedrawerAccordion/SidedrawerAccordionItem.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Item } from '@radix-ui/react-accordion'
+
+type SidedrawerAccordionItemProps = React.ComponentProps<typeof Item>
+
+export const SidedrawerAccordionItem: React.FC<
+  SidedrawerAccordionItemProps
+> = ({ children, value, ...remainingProps }) => (
+  <Item value={value} {...remainingProps}>
```

Comment:
```suggestion
  <Item {...remainingProps}>
```

`value` can be passed through `remainingProps`